hermeto/core/resolver.py

```diff
@@ -65,16 +70,35 @@ def _resolve_packages(request: Request) -> RequestOutput:
     """Run all requested package managers, return their combined output."""
     _supported_package_managers = _package_managers
     requested_types = set(pkg.type for pkg in request.packages)
-    if "dev-package-managers" in request.flags:
-        _supported_package_managers = _package_managers | _dev_package_managers
-    unsupported_types = requested_types - _supported_package_managers.keys()
-    if unsupported_types:
+
+    # Convert x-<pkg> -> <pkg> and track which ones are experimental
+    normalized_types: set[PackageManagerType] = set()
```

Comment:
A readability nitpick: I think this should be better named `requested_normalized_types` since that's what this name appears to refer to. The same applies to `experimental_types`. Also, do you really need a type hint here?